.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: node_js
 node_js:
-- '0.11'
-- '0.10'
+  - 0.10
+  - 0.12
+  - 4
+  - 6
+  - lts/*
```

Comment:
This might look weird but is implemented like this in Travis, which uses NVM: https://github.com/creationix/nvm#long-term-support